code/game/objects/items/clown_items.dm

```diff
@@ -162,7 +162,7 @@
 /obj/item/bikehorn
 	name = "bike horn"
 	desc = "A horn off of a bicycle. Rumour has it that they're made from recycled clowns."
-	icon = 'icons/obj/weapons/horn.dmi'
+	icon = 'icons/obj/art/horn.dmi'
```

Comment:
Why art?

Reply:
because it's a musical thing, and musical things are in /art/